### DISCUSSION TITLE
Rename server

### DISCUSSION THREAD

**Discussion by ValgulNecron** [moghtech/komodo#118] [Q&A] [answered]
Hi i'm ussing docker compose to deploy komodo and it give my server a name like "server-{random-char}" and want to rename it to something like "hetzner server" how can i do this ?

**Comment by mbecker20** [comment, marked as answer]
If you go to the server config, you can scroll all the way to the bottom. There, you will find the Rename Server input.

**Reply by ValgulNecron** [reply]
hi thanks for the response
